src/page/courier/TaskListPage.js

```diff
@@ -8,9 +8,12 @@ import moment from 'moment/min/moment-with-locales'
 import TaskList from '../../components/TaskList'
 import DateSelectHeader from '../../components/DateSelectHeader'
 import { whiteColor } from '../../styles/common'
-import { changedTasks, loadTasksRequest } from "../../store/actions"
 import { translate } from 'react-i18next'
 import { localeDetector } from '../../i18n'
+import {
+  loadTasks, markTaskDone, markTaskFailed,
+  selectTasksList, selectTaskSelectedDate, selectIsTasksLoading,
+} from '../../redux/Tasks'
```

Comment:
Hey,

I like the idea of splitting out things, but I think a better namespacing would be `Courier`, `Customer` and `Restaurant` - so it will stick to the  3 different user kinds.

Reply:
Are you thinking that this namespace would just be reflected in the directory structure, or also the shape of the state?

I think the former is fine, but less sure about the latter.